### DISCUSSION TITLE
데이터 타입이 필요한 이유는 무엇이라고 생각하시나요?

### DISCUSSION THREAD

**Discussion by pipisebastian** [publdaze/js-study#8] [데이터 타입]
- 데이터 타입이 필요한 이유는 무엇이라고 생각하시나요?

**Comment by publdaze** [comment]
> - 데이터 타입이 필요한 이유는 무엇이라고 생각하시나요?
- 값 저장 시 확보할 메모리 공간 크기 결정
- 값 참조 시 확보할 메모리 공간 크기 결정
- 2진수 어떻게 해석할 지 결정

**Comment by pipisebastian** [comment]
>데이터 타입이 필요한 이유는 무엇이라고 생각하시나요? 

1. 값을 저장할때 확보해야할 메모리 공간의 크기를 알아야 함
2. 값을 참조할때 읽어들여야할 메모리 공간의 크기를 알아야 함
3. 메모리 값(2진수)을 어떻게 해석할건지 알아야 함
